etc/test/jest.config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+    // Automatically clear mock calls and instances between every test
+    clearMocks: true,
+
+    // Sets the path that later can be referenced as a <rootDir> token
+    rootDir: '../../',
+
+    // A list of paths to directories that Jest should use to search for files in
+    roots: ['<rootDir>/src'],
+
+    // An array of directory names to be searched recursively up from the requiring module's location
+    moduleDirectories: ['node_modules', 'src/packages'],
+
+    // The path to a module that runs some code to configure or set up the testing framework before each test
+    setupFiles: ['<rootDir>etc/test/enzyme.setup.js'],
+
+    // The glob patterns Jest uses to detect test files
+    testMatch: ['**/__tests__/**/*.spec.js'],
+
+    // A map from regular expressions to paths to transformers.
+    // A transformer is a module that provides a synchronous function for transforming source files.
+    transform: {
+        '^.+\\.jsx?$': 'babel-jest',
+        '^.+\\.scss$': 'jest-css-modules-transform'
+    },
+
+    // An array of glob patterns indicating a set of files for which coverage information should be collected.
+    // If a file matches the specified glob pattern, coverage information will be collected for it even if no tests
+    // exist for this file and it's never required in the test suite.
+    collectCoverageFrom: ['**/*.{js,jsx}'],
+
+    coveragePathIgnorePatterns: ['/node_modules/', 'dummy-components']
```

Comment:
Should this perhaps also include `/example`?

Reply:
Seems like that the only thing that affects coverage score here is `'dummy-components'` pattern, probably because of `roots: ['<rootDir>/src']`. So we can even remove `/node_modules/` :)